src/Jadu/Pulsar/Twig/Extension/HelperOptionsModifierExtension.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace Jadu\Pulsar\Twig\Extension;
+
+class HelperOptionsModifierExtension extends \Twig_Extension
+{
+	/**
+	 * Name of this extension
+	 *
+	 * @return string
+	 */
+	public function getName()
+    {
+        return 'modify_options';
+    }
+
+    public function getFunctions()
+    {
+        return array(
+            new \Twig_SimpleFunction('modify_options', array($this, 'modifyOptions'))
+        );
+    }
+
+    /**
+	 * Returns a modified options array including error flag, error and help guids
+	 *
+	 * @param  array    $options    The form helper options
+	 * @return array
+	 */
+    public function modifyOptions($options)
+    {
+        // If we don't have $options, abandon ship
+        if ($options === null) {
+            return false;
```

Comment:
This return value should probably be specified in the docblock `@return array|boolean`

Reply:
Added